src/components/views/sul-people/sul-people-table-view.tsx

```diff
@@ -238,11 +240,17 @@ const SulPeopleTableView = ({items, hasHeading}: Props) => {
                         <EnvelopeIcon title="Email" width={20} className="mr-6 inline-block" />
                         {item.email}
                       </Link>
-                    </>
+                    </div>
                   )}
                 </Td>
                 <Td className="block w-auto px-0 py-16 text-center md:border-b md:border-black-40 md:px-9 md:py-16 md:text-left lg:table-cell lg:w-1/5">
-                  {item.libCalId && <LibCal libcalId={item.libCalId} srText={item.title} />}
+                  {item.libCalId ? (
+                    <div className="pb-16 lg:pb-0">
+                      <LibCal libcalId={item.libCalId} srText={item.title} />
+                    </div>
+                  ) : (
+                    ""
+                  )}
```

Comment:
No need for an empty "else" condition.

Reply:
I have the empty else because without it, it renders out a `&nbsp;` which leaves vertical space.  Is there another way to get rid of it?
<img width="419" alt="Screenshot 2024-08-22 at 9 20 16 AM" src="https://github.com/user-attachments/assets/76a94192-bafa-4b4f-84d3-6067800a151b">
<img width="460" alt="Screenshot 2024-08-22 at 9 18 20 AM" src="https://github.com/user-attachments/assets/7407784a-242f-4395-b3c2-6f6ed6b5f11c">

Reply:
This is on the smallest mobile view, btw.